smarts/core/remote_agent_buffer.py

```diff
@@ -209,13 +171,32 @@ def _try_to_acquire_remote_agent(self):
         return remote_agent
 
     def acquire_remote_agent(self, retries=3) -> RemoteAgent:
-        for i in range(retries):
+        for ii in range(retries):
```

Comment:
`ii` stands for?

Reply:
I inherited the habit of using `ii`, `jj`, `kk`, `ll`, etc, for loop variables from my digital signal processing days. This notation is pretty common to denote iterator variables in digital communication. Using variables with repeated letter also helps to prevent silly mistakes while coding.

I have now replaced the line `for ii in range(retries):` to `for retry in range(retries):` making it more readable and Pythonic.